geomesa-nifi-processors/src/main/scala/org/geomesa/nifi/geo/AbstractGeoIngestProcessor.scala

```diff
@@ -169,17 +167,20 @@ abstract class AbstractGeoIngestProcessor(dataStoreProperties: Seq[PropertyDescr
   }
 
   @OnRemoved
-  @OnDisabled
+  @OnStopped
+  @OnShutdown
```

Comment:
Do we want this to run OnDisabled?  Who else has read through these annotations before?

Reply:
OnDisabled only applies to ControllerServices: https://github.com/apache/nifi/blob/master/nifi-api/src/main/java/org/apache/nifi/annotation/lifecycle/OnDisabled.java#L47-L48